plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -543,87 +547,88 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.JUNGLE_TRAPDOOR, 0x9D7150);
         put(Blocks.JUNGLE_WALL_SIGN, 0xA17351);
         put(Blocks.JUNGLE_WOOD, 0x554419);
-        put(Blocks.KELP, 0x578B2C);
         put(Blocks.KELP_PLANT, 0x57802A);
+        put(Blocks.KELP, 0x578B2C);
         put(Blocks.LADDER, 0x000000);
         put(Blocks.LANTERN, 0x6A5B54);
         put(Blocks.LAPIS_BLOCK, 0x1F438C);
         put(Blocks.LAPIS_ORE, 0x68758F);
         put(Blocks.LARGE_AMETHYST_BUD, 0xA582CD);
-        put(Blocks.LARGE_FERN, 0x000000);
-        put(Blocks.LAVA, 0xD45A12);
+        put(Blocks.LARGE_FERN, 0x7D4536);
```

Comment:
Same as the grass, this makes some parts of the map look like green static, so it should be invisible